### DISCUSSION TITLE
Guia de Inicio de colaboradores

### DISCUSSION THREAD

**Discussion by DouglasLopez** [fhcn-io/community-accelerator#9] [Ideas]
Colaboremos en definir algunos aspectos basicos de la guia de inicio de colaboradores

**Comment by DouglasLopez** [comment]
Tengo algunas dudas que me gustaria definir para trabajar en este issue podemos discutirlas aca ? gracias por su colaboracion @ramrodo @ar4mirez

**Comment by DouglasLopez** [comment]
Duda numero 1 ? sujetamos el sitio a crear comunidades de la cncf? o lo dejamos de un aspecto mas amplio

**Reply by ar4mirez** [reply]
Yo sugiero no cerrarnos solo a la CNCF asi poder apoyar a mas comunidades tal vez la pregunta es acotarlo a comunidades tecnologicas?

**Comment by DouglasLopez** [comment]
Cuantos approvals a los  PR requerimos? 2 o 1

**Reply by electrocucaracha** [reply]
Quizá dos de distintas empresas para mantener neutralidad

**Reply by DouglasLopez** [reply]
Bueno de hecho serian colaboradores pero si me parece, lo que me preocupa de las dos aprobaciones es que necesitariamos mas colaboradores para que no se vuelva cuello de botella

**Comment by DouglasLopez** [comment]
Sugerimos o definimos una guia de estilos o lo dejamos libre?

**Reply by electrocucaracha** [reply]
Por el momento creo que deberíamos utilizar los estilos propuestos por algun linter, sugiero [super-linter](https://github.com/github/super-linter)

**Reply by ar4mirez** [reply]
Creo que es mejor sugerir ya que muchos vienen preguntando por guias.

**Reply by DouglasLopez** [reply]
Con la guia de estilos me refiero a que cada documento cumpla con ciertas secciones basicas para que tenga cierto orden ejemplo podriamos poner estos templates https://thegooddocsproject.dev/

**Reply by ar4mirez** [reply]
No estoy familiarizado con el proyecto que mencionas pero me hace mucho sentido sugerir basado en un buen estandar.

**Reply by adawolfs** [reply]
Hice alguna investigacion y creo que se ajusta bastante bien en especial el template de "how-to".
https://gitlab.com/tgdp/templates/-/blob/main/how-to/template-how-to.md

**Comment by adawolfs** [comment]
Para el framework web sugiero esta opcion: https://docusaurus.io/docs , me gusto el estilo, funciona con markdown y tiene un buscador y navbar integrado, entre otros es el que usa jest para sus docs.
https://jestjs.io/docs/getting-started 
https://github.com/jestjs/jest/blob/main/docs/GettingStarted.md

**Reply by ar4mirez** [reply]
Conozco DocuSaurus y si es muy bueno, la otra que conozco es Hugo (gohugo.io)
